shared/settings/nav/index.desktop.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import React, {Component} from 'react'
+import {globalStyles, globalColors, globalMargins} from '../../styles'
+import {Box, Badge, ClickableBox, Text} from '../../common-adapters'
+
+import type {Props, BannerType, SettingsItem as SettingsItemType} from './index'
+
+function Banner ({element, type}: {element: React$Element<*>, type: BannerType}) {
+  return (
+    <Box style={{...commonBannerStyle, ...variantBannerStyle[type]}}>
```

Comment:
Yeah, agreed. It would probably be worth breaking out the banner into a reusable component at some point -- I believe this is the 3rd implementation of it I've seen around.